e2e/start-in-background.e2e.spec.ts

```diff
@@ -3,38 +3,33 @@ import path from 'path';
 import { test, expect, ElectronApplication } from '@playwright/test';
 
 import {
-  createDefaultSettings, packageLogs, startRancherDesktop, teardownApp, tool,
+  createDefaultSettings, reportAsset, startRancherDesktop, teardownApp, tool,
 } from './utils/TestUtils';
 
 /**
  * Using test.describe.serial make the test execute step by step, as described on each `test()` order
  * Playwright executes test in parallel by default and it will not work for our app backend loading process.
  * */
 test.describe.serial('startInBackground setting', () => {
-  test.afterAll(async() => {
-    await packageLogs(__filename);
-  });
-
   test('window should appear when startInBackground is false', async() => {
     createDefaultSettings({ application: { startInBackground: false } });
-    const electronApp = await startRancherDesktop(__filename);
+    const logName = `${ __filename }-startInBackgroundFalse`;
+    const electronApp = await startRancherDesktop(__filename, { logName });
 
     await expect(checkWindowOpened(electronApp)).resolves.toBe(true);
-    const tracePath = path.join(__dirname, 'reports', `${ path.basename(__filename) }-startInBackgroundFalse.zip`);
-
-    await electronApp.context().tracing.stop({ path: tracePath });
+    await electronApp.context().tracing.stop({ path: reportAsset(logName, 'trace') });
     await teardownApp(electronApp);
```

Comment:
i'm wondering why are we using `teardownApp` alone when `teardown` calls `teardownApp` and `reportAsset`?

Reply:
It's like that before the change, but I _think_ it's because we're testing shutdown behaviour here so we expect the app to have _already_ quit. But you're right, the only difference is some logging…